docs/gallery/interactive/fast_fourier_transform.ipynb

```diff
@@ -0,0 +1,264 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Explanations\n",
+    "\n",
+    "* `'none'`:\n",
+    "        Use the spectrum as it is. This norm is to be used for energy signals such as impulse responses (IRs). In this case the spectrum is independent of the signal length. For power signals, the spectrum depends on the number of samples (\"longer signal = more energy\"), so different normalizations are appropriate.\n",
+    "* `'unitary'`:\n",
+    "        Multiply the spectrum with a factor of 2 in order to represent power related measures correctly (e.g., the amplitude or RMS-value, see below). **All following normalizations make use of this.**\n",
+    "* `'amplitude'`:\n",
+    "        Normalize the spectrum to show the amplitude of the pure tone components contained in a signal. If the signal is a sine with an amplitude of 1, the spectrum will have an absolute value of 1 (0 dB) at the frequency of the sine.\n",
+    "* `'rms'`:\n",
+    "        Normalize the spectrum to show the RMS value of the pure tone components contained in a signal. If the signal is a sine with an amplitude of 1, the spectrum will have an absolute value of $1/\\sqrt{2}$ (-3 dB) at the frequency of the sine.\n",
+    "* `'power'`:\n",
+    "        In a dB representation, it equals the `'rms'` normalization describing a power quantity. For stochastic broadband signals, this results in a dependence of the magnitude on the sampling rate as these are defined by a constant power density (see `'psd'`).\n",
+    "* `'psd'`:\n",
+    "        Using this normalization, signals are represented as *power densities* (e.g. in Pa²/Hz), leading to a meaningful representation for broadband stochastic signals independently of the sampling rate."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Overview: Appropriate FFT normalizations\n",
+    "\n",
+    "| Signal type            | Variation                              | Normalization          |\n",
+    "|------------------------|----------------------------------------|------------------------|\n",
+    "| Energy                 | Impulse responses / transfer functions | `'none'`               |\n",
+    "| Power                  | Discrete tones                         | `'amplitude'`, `'rms'`, `'power'` |\n",
+    "| Power                  | Broadband stochastic signals           | `'psd'`              |"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Examples\n",
+    "Four signals with a length of 100 samples and a sampling rate of 10 kHz are used for illustration."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import pyfar as pf\n",
+    "import numpy as np\n",
+    "import matplotlib.pyplot as plt\n",
+    "\n",
```

Comment:
To make it future proof you could ad `%matplotlib inline` to make sure all figures are rendered if we were to add a second figure at some point.

Reply:
I like to keep it as clean as possible for now and leave this up to the future author :)